index.html

```diff
@@ -2271,7 +2271,7 @@ <h4>Additional information</h4>
 		<section> <!-- Web Development -->
 			<h2>Web Development</h2>
 			<p><strong>Sustainable web design and development practices at the front-end and back-end often intersect with best practices, unlocking numerous benefits for people and the planet alike.</strong></p>
```

Comment:
Perhaps a personal preference - "on" rather than "at" reads better in this sentences.

"Sustainable web design and development practices _on_ the front-end and back-end ..."